lib/genesis_collector/collector.rb

```diff
@@ -120,6 +123,10 @@ def collect_memories
       end.compact
     end
 
+    def collect_raids
+      collect_megaraid
```

Comment:
I'm wondering if we should make a function `megaraid?` so we can do an if else block we we want to collect data about non-megaraids?. If not just directly call collect_megaraid above